cmd/kyma/provision/minikube/cmd.go

```diff
@@ -73,6 +72,7 @@ func NewCmd(o *Options) *cobra.Command {
 	cmd.Flags().StringVar(&o.Profile, "profile", "", "Specifies the Minikube profile.")
 	cmd.Flags().DurationVar(&o.Timeout, "timeout", 5*time.Minute, `Maximum time during which the provisioning takes place, where "0" means "infinite". Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".`)
 	cmd.Flags().BoolVar(&o.UseVPNKitSock, "use-hyperkit-vpnkit-sock", false, `Uses vpnkit sock provided by Docker. This is useful when DNS Port (53) is being used by some other program like dns-proxy (eg. provided by Cisco Umbrella. This flag works only on Mac OS).`)
+	cmd.Flags().StringVarP(&o.KubernetesVersion, "kube-version", "k", "1.16.3", "Kubernetes version of the cluster.")
```

Comment:
Is it possible to use 1.16 as default? and minikube will just fetch latest?

Reply:
I have tried to do this, but you get an error if `Patch` is not specified: `X Unable to parse "v1.16": No Major.Minor.Patch elements found`.

So, I used the version which was used before: https://github.com/kyma-project/cli/pull/623/files#diff-6aae7007b787b0bc78b9694e93aeee817291de1aba6d3b299953434d7019810cL23

Reply:
ok, so please update to latest supported patch version directly, makes no sense to continue with an outdated version

Reply:
1.16.15 is used as the default version now